codeforboston/vaccinatema
# Enable static lookup for zip codes in MA.

Per @mzagaja's suggestion, this PR enables us to look up the latitude and longitude of zip codes without hitting the Geocoder API (which is getting expensive).

To get these latitudes and longitudes, I created a Google Sheet with all MA zip codes, created a custom formula (`GEOCODE, below`), and then converted the results to JSON.
```
function GEOCODE(zipCode) {
  var response = Maps.newGeocoder().geocode(zipCode);
  var location = response.results[0].geometry.location;
  return `${location.lat}, ${location.lng}`;
}
```

### Test Plan:
* Add temporary logging or a breakpoint to `return zipCodes[req.body.zipCode];` in `distance-utils`.
* Search for a zip code in MA. See logging / breakpoint hit. Ensure results match those on the live site.
* Search for a zip code outside MA. See that results are still returned as expected.
